pkg/volume/aws_ebs/attacher.go

```diff
@@ -249,7 +249,7 @@ func (plugin *awsElasticBlockStorePlugin) NewDetacher() (volume.Detacher, error)
 }
 
 func (detacher *awsElasticBlockStoreDetacher) Detach(volumeName string, nodeName types.NodeName) error {
-	volumeID := aws.KubernetesVolumeID(path.Base(volumeName))
+	volumeID := aws.KubernetesVolumeID(filepath.Base(volumeName))
```

Comment:
I do not think we need to use filepath here because this is strictly for getting volume-id from a string formatted like `aws://us-east1-a/vol-abcde` . The runtime/os dependency of path does not come into picture here.

Reply:
Would it cause an issue if we do? If the dependency doesn't come into picture, `filepath` should behave the same as `path`?

Reply:
It can actually. We do not want controller-manager running on Windows to use "\" as path separator for this particular call, because no matter which platform kubernetes is running this string will always be formatted with "/"

Reply:
Ah, gotcha. Alright, I'll change that to path.